infrastructure/azure.tf

```diff
@@ -663,7 +649,6 @@ resource "null_resource" "azurerm_app_service_portal" {
       "ts-node ${var.app_service_portal_provisioner}",
       "--environment ${var.environment}",
       "--azurerm_resource_group ${azurerm_resource_group.azurerm_resource_group.name}",
-      "--azurerm_functionapp ${azurerm_function_app.azurerm_function_app.name}",
```

Comment:
is this related to sendgrid?

Reply:
it isn't (ref. https://github.com/teamdigitale/digital-citizenship/pull/77#issue-179668607)

Reply:
then change the name of this PR

Reply:
done.